aws/resource_aws_kinesis_firehose_delivery_stream.go

```diff
@@ -321,6 +321,10 @@ func flattenKinesisFirehoseDeliveryStream(d *schema.ResourceData, s *firehose.De
 				"retry_duration":             *destination.SplunkDestinationDescription.RetryOptions.DurationInSeconds,
 			}
 
+			if v := destination.SplunkDestinationDescription.ProcessingConfiguration; v != nil {
+				splunkConfiguration["processing_configuration"] = v
```

Comment:
I missed this before merging. This should be `= flattenProcessingConfiguration(*v, roleArn)`. I will fix this in an upcoming PR.

Reply:
I was wondering about that, where is RoleArn supposed to come from in the Splunk Configuration?

Reply:
Its only used as a default value to filter out a difference if it is undefined in the Terraform configuration under the parameters. In this case, passing it `""` is fine. I have a PR almost ready that includes this fix.